main/UseCases/HRSystem.java

```diff
@@ -21,19 +21,10 @@ method makeInternToPrompt which returns the interns in the intern_list as a form
 public class HRSystem implements Serializable {
     public static final int FINAL_MONTH = 5;
 
-    private ArrayList<HiredIntern> hiredInternList;
+    private final ArrayList<HiredIntern> hiredInternList;
 
```

Comment:
is it possible to make the two lists one? Or refactor methods like getInterviewInternNames and getHiredInternNames to be one method, which could take a string as a parameter. And use Enums to do the if statement (see how GameManager used enums for the month)

Reply:
mm I see what you mean, in terms of making the lists one I think it'll affect the logic a lot for the code dependent on them(like interview level which assumes its only either interview or hired interns not them all together) so refactoring the methods that do similar things would be better (but I also am ??? about the logic for that as well since they require separate private variables). I'll figure that out for the next pull request if that's okay?

Reply:
yep!